sky/resources.py

```diff
@@ -1117,10 +1117,75 @@ def get_required_cloud_features(
         return features
 
     @classmethod
-    def from_yaml_config(cls, config: Optional[Dict[str, str]]) -> 'Resources':
+    def from_yaml_config(
+        cls, config: Optional[Dict[str, Any]]
+    ) -> Union[Set['Resources'], List['Resources']]:
         if config is None:
-            return Resources()
+            return {Resources()}
+
+        def _override_resources(base_resource_config,
+                                override_configs) -> List[Resources]:
+            resources_list = []
+            for override_config in override_configs:
+                new_resource_config = base_resource_config.copy()
+                new_resource_config.update(override_config)
+                resources_list.append(
+                    Resources._from_yaml_config_single(new_resource_config))
+            return resources_list
+
+        config = config.copy()
+        any_of_configs = config.pop('any_of', None)
+        ordered_configs = config.pop('ordered', None)
+        if any_of_configs and ordered_configs:
+            with ux_utils.print_exception_no_traceback():
+                raise ValueError(
+                    'Cannot specify both "any_of" and "ordered" in resources.')
+
+        # Parse resources.accelerators field.
```

Comment:
Could we use any_of under accelerators to represent multiple accelerators..?

Reply:
Currently, no. We probably want to keep the API simple, instead of have multiple levels of the same field. Wdyt?

Reply:
I think using `{}` interface is cleaner if we have more accelerators.